### ISSUE TITLE
I registered my dog, the generated .pdf contains 3 instances of the same information

### ISSUE THREAD

**Issue by 0xfd000000** [LilithWittmann/formularium#21]
Hi!

Sorry if this is intended behavior or a known issue...

On https://formularium.verdrusssache.de/ I filled the form Hundehaltung/Anmeldung and the generated .pdf contains three copies of

> Grund für die Meldung
> ...
> Angaben zur Person
> ...

see attachment: [file.pdf](https://github.com/LilithWittmann/formularium/files/6176407/file.pdf)

**Comment by LilithWittmann** [comment]
Hey, did you generate that in the Editor or in the form itself? (That's actually a known bug in the editor if you skip a lot between form pages and don't use the rest button)

**Comment by 0xfd000000** [comment]
This was generated from the form itself and I don't thin I went back between pages, but I'll quickly try to reproduce...

**Comment by 0xfd000000** [comment]
You're right, I could not reproduce it now. I *did* go between the first form page (the one where i choose Anmeldung...) and the second page containing the fields in question the first time I filled it.

But in both cases it was the form itself, not the editor.

**Comment by LilithWittmann** [comment]
Ok, thank you so much ❤️ 
That actually helps me to understand what's going wrong there: For the different paths, we generate different JSON sections that are marked as visited and then are added to the pdf report.

The good thing: this is actually already kind of fixed for the forms that are generated with the new form section editor.

But I will keep it open and will fix it anyway because there are actually good reasons why you want to use blockly for form sections in the future too 😊.
